test-data/apply-configuration/overall/oauth-server-creation-minimal/test.yaml

```
@@ -0,0 +1,37 @@
+  The purpose of this test is to show which input resources are required to run the operator and the controller.
+
+  input-dir:
+  - config.openshift.io/clusterversions: required to "start the operator" (CreateOperatorStarter/prepareOauthOperator)
+  - config.openshift.io/infrastructures: not strictly required (CreateOperatorStarter/prepareOauthAPIServerOperator), but if missing, SNO is assumed and a warning is produced
+  - config.openshift.io/authentications: required by the controller, specifically the "cluster" object
+  - operator.openshift.io/authentications: required by the controller, specifically the "cluster" object
+  - openshift-authentication/route.openshift.io/routes/oauth-openshift: required by the controller, single-item informer
+  - core/nodes: (i think) required, otherwise the replica count on the deployment will be 0
+  dependencies specified directly on the deployment:
+  configmaps:
+  - core/configmaps/openshift-authentication/audit: required, otherwise the pod won't start 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)